starters/vue3-apollo-quasar/src/components/NumberCounter/NumberCounter.vue

```diff
@@ -36,15 +36,42 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
+import { computed, defineComponent } from 'vue';
 
 export default defineComponent({
   name: 'NumberCounter',
 });
 </script>
 
 <script lang="ts" setup>
-import { useCounterStore } from 'src/stores';
+import { counts } from 'src/variables/counts';
```

Comment:
I think this is an unnecessary abstraction, we can move this logic into the number counter component. Or, if this is meant to serve the same purpose as a counter store, then we should be able have it within this component (in which case it isn't global)

Reply:
I think the abstraction is okay.

Reply:
The way I see it, we have 2 options:

- The abstraction is unnecessary if the mutation logic is coupled to the `NumberCounter` component. In this case move the logic into the `NumberCounter` component. This negates the point of using apollo for global state though. No need to do 50-50 with the implementation

- The mutation logic gets moved into the abstraction and out of `NumberCounter` to make it truly global functionality.

Reply:
I think that's what i have done. thansk